tealium/__init__.py

```diff
@@ -99,40 +105,89 @@ def enable(self):
             statusResult = False
         return statusResult
 
-    def trackEvent(self, title, data=None):
-
+    def trackEvent(self, title, eventType=None, data=None, tealiumCallback=None):
         '''
             Call with a title and optional dictionary to track an event in
             stream.
 
         Args:
             Title (String): A title for the event being tracked.
 
+            EventType(String): (EVENT_TYPE_CONVERSION,
+                                    EVENT_TYPE_DERIVED,
+                                    EVENT_TYPE_ACTIVITY,
+                                    EVENT_TYPE_VIEW,
+                                    EVENT_TYPE_INTERACTION )
+
             Data (optional dictionary): Add any additional data sources for the
-            corresponding event.
+                    corresponding event.
+
+            Callback(optional callback):
+                Args:
+                    Info : Optional dictionary of data from call (i.e.
+                        response headers, encoded Url String etc.)
+                    Success: Whether completion succeeded or encountered
+                        a failure.
+                    Error: Error encounter, if any.
         '''
         randomNumber = self.generateRandomNumber()
+        if eventType is None:
+            eventType = EVENT_TYPE_ACTIVITY
+
         TRACK_URL = '{}tealium_account={}&tealium_profile={}&tealium_'\
                     'environment={}&tealium_vid={}&tealium_random='\
                     '{}&event_name={}&tealium_timestamp_epoch={}' \
                     '&tealium_session_id={}&tealium_'\
-                    'visitor_id={}'.format(self.T_BASE_URL, self.account,
-                                           self.profile, self.environment,
-                                           self.uuid, randomNumber, title,
-                                           self.generateTimeStamp(), sessionId,
-                                           self.uuid)
+                    'visitor_id={}&tealium_event={}&'\
+                    'tealium_event_type={}'.format(self.T_BASE_URL,
+                                                   self.account,
+                                                   self.profile,
+                                                   self.environment,
+                                                   self.uuid,
+                                                   randomNumber,
+                                                   title,
+                                                   self.generateTimeStamp(),
+                                                   sessionId,
+                                                   self.uuid,
+                                                   title,
+                                                   eventType)
         if data is not None:
             for key in data:
                 if sys.version_info[:3] >= (3, 0):
```

Comment:
Might be worth setting the version check into a variable near the top of the script, instead of dynamically checking each time.